api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/EriDSDDataset.java

```diff
@@ -526,14 +526,92 @@ public DataSetDefinition constructEriDSDDataset() {
             mappings),
         "");
 
+    dsd.addColumn(
+        "N5T",
+        "DSD N5 Total",
+        EptsReportUtils.map(
+            eptsGeneralIndicator.getIndicator(
+                "N5T",
+                EptsReportUtils.map(
+                    eriDSDCohortQueries.getPatientsWhoAreActiveAndParticipatingInAccessionClubs(),
+                    mappings)),
+            mappings),
+        "");
+    dsd.addColumn(
+        "N5SST",
+        "DSD N5 Stable subtotal",
+        EptsReportUtils.map(
+            eptsGeneralIndicator.getIndicator(
+                "N5SST",
+                EptsReportUtils.map(
+                    eriDSDCohortQueries
+                        .getPatientsWhoAreActiveParticipatingInAccessionClubsAndStable(),
+                    mappings)),
+            mappings),
+        "");
+    dsd.addColumn(
+        "N5SNPNBA",
+        "DSD N5 Stable Non-pregnant and Non-Breastfeeding Adults (>=15)",
+        EptsReportUtils.map(
+            eptsGeneralIndicator.getIndicator(
+                "N5SNPNBA",
+                EptsReportUtils.map(
+                    eriDSDCohortQueries.getN5StableNonPregnantNonBreastfeeding(), mappings)),
+            mappings),
+        "age=15+");
+    addRow(
+        dsd,
+        "N5SNPNBC",
+        " DSD N5 Stable Non-pregnant and Non-Breastfeeding Children (2-4, 5-9, 10-14)",
+        EptsReportUtils.map(
+            eptsGeneralIndicator.getIndicator(
+                "N5SNPNBC",
+                EptsReportUtils.map(
+                    eriDSDCohortQueries.getN5StableNonPregnantNonBreastfeeding(), mappings)),
+            mappings),
+        getChildrenColumn());
+    dsd.addColumn(
+        "N5UST",
+        "DSD N5 Unstable subtotal",
+        EptsReportUtils.map(
+            eptsGeneralIndicator.getIndicator(
+                "N5UST",
+                EptsReportUtils.map(
+                    eriDSDCohortQueries
+                        .getPatientsWhoAreActiveParticipatingInAccessionClubsAndUnstable(),
+                    mappings)),
+            mappings),
+        "");
+    dsd.addColumn(
+        "N5UNPNBA",
+        "DSD N5 Unstable Non-pregnant and Non-Breastfeeding Adults (>=15)",
+        EptsReportUtils.map(
+            eptsGeneralIndicator.getIndicator(
+                "N5UNPNBA",
+                EptsReportUtils.map(
+                    eriDSDCohortQueries.getN5UnstableNonPregnantNonBreastfeeding(), mappings)),
+            mappings),
+        "age=15+");
+    addRow(
+        dsd,
+        "N5UNPNBC",
+        " DSD N5 Unstable Non-pregnant and Non-Breastfeeding Children (2-4, 5-9, 10-14)",
+        EptsReportUtils.map(
+            eptsGeneralIndicator.getIndicator(
+                "N5UNPNBC",
+                EptsReportUtils.map(
+                    eriDSDCohortQueries.getN5UnstableNonPregnantNonBreastfeeding(), mappings)),
```

Comment:
If you address the first comment. then you can easily replace this method call with "eriDSDCohortQueries.getPatientsWhoAreActiveParticipatingInAccessionClubsAndUnstable()"